AllASPFuncs.py

```diff
@@ -730,7 +730,10 @@ def radec2point(RA, DEC, filt, start = None, end = None, path = '/cwork/mat90/Ro
     allRA = f['RA']
     allDEC = f['DEC']
 
-    dist = np.sqrt((allRA - RA)**2 + (allDEC - DEC)**2)
+
+    all_coords = SkyCoord(allRA*u.deg, allDEC*u.deg, frame='icrs')
```

Comment:
Perhaps "pointing_sca_coords" would be a good opportunity for helpful documentation through variable names.